### DISCUSSION TITLE
http://127.0.0.1:7860/?__theme=dark

### DISCUSSION THREAD

**Discussion by adrianpuiu** [seruva19/kubin#68] [General]
add to launch options : 

http://127.0.0.1:7860/?__theme=dark

**Comment by seruva19** [comment]
You mean - mention it in docs (wiki)?

**Comment by adrianpuiu** [comment]
yep

**Reply by adrianpuiu** [reply]
PS : i messed up last training with 100 images,  but was a good lesson not to use auto resize and crop ..the pb is being on linux im not able to use Photoshop ... and i don't know any other good tool for batch cropping .. ive tried birme website and gave bunch of errors due to long image namings
